plugins/@grouparoo/sqlite/src/lib/export/exportProfile.ts

```diff
@@ -37,11 +37,11 @@ export const exportProfile: ExportProfilePluginMethod = async ({
         );
       }
       // delete
-      const query = `DELETE FROM ${table} WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
+      const query = `DELETE FROM "${table}" WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
       validateQuery(query);
       await connection.asyncQuery(query);
     } else if (newProfileProperties[primaryKey]) {
-      const query = `SELECT * FROM ${table} WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
+      const query = `SELECT * FROM "${table}" WHERE ${primaryKey} = ${newProfileProperties[primaryKey]}`;
```

Comment:
Does `primaryKey` need to be quoted here too?  What if the column name was `"user id"` or something like that?

> SELECT * FROM "users" where user id = 1